generated_chains.go

```diff
@@ -0,0 +1,127 @@
+	UNNAMED_90000007                        = Chain{EvmChainID: 90000007, Selector: 8412806778050735057, Name: "90000007"}
+	UNNAMED_90000008                        = Chain{EvmChainID: 90000008, Selector: 4066443121807923198, Name: "90000008"}
+	UNNAMED_90000009                        = Chain{EvmChainID: 90000009, Selector: 6747736380229414777, Name: "90000009"}
+	UNNAMED_90000010                        = Chain{EvmChainID: 90000010, Selector: 8694984074292254623, Name: "90000010"}
+	UNNAMED_90000011                        = Chain{EvmChainID: 90000011, Selector: 328334718812072308, Name: "90000011"}
+	UNNAMED_90000012                        = Chain{EvmChainID: 90000012, Selector: 7715160997071429212, Name: "90000012"}
+	UNNAMED_90000013                        = Chain{EvmChainID: 90000013, Selector: 3574539439524578558, Name: "90000013"}
+	UNNAMED_90000014                        = Chain{EvmChainID: 90000014, Selector: 4543928599863227519, Name: "90000014"}
+	UNNAMED_90000015                        = Chain{EvmChainID: 90000015, Selector: 6443235356619661032, Name: "90000015"}
+	UNNAMED_90000016                        = Chain{EvmChainID: 90000016, Selector: 13087962012083037329, Name: "90000016"}
+	UNNAMED_90000017                        = Chain{EvmChainID: 90000017, Selector: 11985232338641871056, Name: "90000017"}
+	UNNAMED_90000018                        = Chain{EvmChainID: 90000018, Selector: 7777066535355430289, Name: "90000018"}
+	UNNAMED_90000019                        = Chain{EvmChainID: 90000019, Selector: 1273605685587320666, Name: "90000019"}
+	UNNAMED_90000020                        = Chain{EvmChainID: 90000020, Selector: 17810359353458878177, Name: "90000020"}
+	WEMIX_MAINNET                           = Chain{EvmChainID: 1111, Selector: 5142893604156789321, Name: "wemix-mainnet"}
+	WEMIX_TESTNET                           = Chain{EvmChainID: 1112, Selector: 9284632837123596123, Name: "wemix-testnet"}
+)
+
+var ALL = []Chain{
+	AVALANCHE_MAINNET,
+	AVALANCHE_TESTNET_FUJI,
+	BINANCE_SMART_CHAIN_MAINNET,
+	BINANCE_SMART_CHAIN_TESTNET,
+	BITTORRENT_CHAIN_MAINNET,
+	BITTORRENT_CHAIN_TESTNET,
+	ETHEREUM_MAINNET,
+	ETHEREUM_MAINNET_ARBITRUM_1,
+	ETHEREUM_MAINNET_BASE_1,
+	ETHEREUM_MAINNET_KROMA_1,
+	ETHEREUM_MAINNET_MANTLE_1,
+	ETHEREUM_MAINNET_OPTIMISM_1,
+	ETHEREUM_MAINNET_POLYGON_ZKEVM_1,
+	ETHEREUM_MAINNET_SCROLL_1,
+	ETHEREUM_TESTNET_GOERLI_ARBITRUM_1,
+	ETHEREUM_TESTNET_GOERLI_BASE_1,
+	ETHEREUM_TESTNET_GOERLI_MANTLE_1,
+	ETHEREUM_TESTNET_GOERLI_OPTIMISM_1,
+	ETHEREUM_TESTNET_GOERLI_POLYGON_ZKEVM_1,
+	ETHEREUM_TESTNET_GOERLI_ZKSYNC_1,
+	ETHEREUM_TESTNET_SEPOLIA,
+	ETHEREUM_TESTNET_SEPOLIA_ARBITRUM_1,
+	ETHEREUM_TESTNET_SEPOLIA_BASE_1,
+	ETHEREUM_TESTNET_SEPOLIA_KROMA_1,
+	ETHEREUM_TESTNET_SEPOLIA_OPTIMISM_1,
+	ETHEREUM_TESTNET_SEPOLIA_SCROLL_1,
+	KAVA_MAINNET,
+	KAVA_TESTNET,
+	POLYGON_MAINNET,
+	POLYGON_TESTNET_MUMBAI,
+	UNNAMED_1000,
```

Comment:
Hmm, I would say that either `CHAIN_1000` or `TEST_1000` would be better than unnamed. Leaning towards `CHAIN_` prefix

Reply:
Agree, TEST_ seems appropriate.